tests/testthat/test-rd-family.R

```diff
@@ -123,3 +123,29 @@ test_that("custom family prefixes can be set", {
 
   expect_match(out$get_value("seealso"), "^Custom prefix:")
 })
+
+test_that("careful ordering", {
+  # Can't use foo1 because it's used as class in another test
```

Comment:
Is it an S4 class? It might need a `on.exit(removeClass("foo1"))` call.

Reply:
No, unfortunately, #1564 does not change the behavior here.

Reply:
Happy to adapt if #1566 or a better version gets in.